src/main/java/lingogo/logic/commands/SlideshowCommand.java

```diff
@@ -0,0 +1,30 @@
+package lingogo.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import lingogo.commons.core.Messages;
+import lingogo.logic.commands.exceptions.CommandException;
+import lingogo.model.Model;
+
+public class SlideshowCommand extends Command {
+
+    public static final String COMMAND_WORD = "slideshow";
+    public static final String COMMAND_DESCRIPTION = "Starts the slideshow with the current filtered list of"
+            + "flashcards";
+    public static final String COMMAND_USAGE = "slideshow";
+    public static final String COMMAND_EXAMPLES = "slideshow";
+    public static final String MESSAGE_SUCCESS = "Slideshow started!";
```

Comment:
can include other commands for slideshow, maybe like
"Slideshow started!\n Enter `previous` to ..., `next` to ..., `stop` to ..."